examples/00_helloservice/generated/src/private/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+list(APPEND generated_SRC
```

Comment:
Maybe `generated_SRC` can be named with a prefix, like `00_generated_SRC`. That way we dont have to unset the variables. Also having separated variables for each project can be good for debugging purpose

Reply:
Resolved.